src/components/Genomes/Cobs/Results.tsx

```diff
@@ -50,53 +69,45 @@ const Results: React.FC<ResultsProps> = ({
     },
     {
       Header: 'Genome length',
-      accessor: 'mgnify.attributes.length',
+      accessor: 'mgnify.length',
     },
     {
       Header: 'Num. contigs',
-      accessor: 'mgnify.attributes.num-contigs',
+      accessor: 'mgnify.num_contigs',
     },
     {
       Header: 'Genome completeness',
-      accessor: 'mgnify.attributes.completeness',
+      accessor: 'mgnify.completeness',
     },
     {
       Header: 'Genome contamination',
-      accessor: 'mgnify.attributes.contamination',
+      accessor: 'mgnify.contamination',
     },
     {
       Header: 'Geographic origin',
-      accessor: 'mgnify.attributes.geographic-origin',
+      accessor: 'mgnify.geographic_origin',
     },
     {
       Header: 'Num. K-mers in query',
-      accessor: 'bigsi.num_kmers',
+      accessor: 'cobs.num_kmers',
     },
     {
       Header: 'Num. K-mers found in genome',
-      accessor: 'bigsi.num_kmers_found',
+      accessor: 'cobs.num_kmers_found',
     },
     {
       Header: '% K-mers found',
-      accessor: 'bigsi.percent_kmers_found',
-    },
-    {
-      Header: 'BLAST score (log p)',
-      accessor: (result) => ({
-        score: result.bigsi.score,
-        pvalue: result.bigsi.log_pvalue,
-      }),
-      Cell: ({ cell }) => `${cell.value.score} (${cell.value.pvalue})`,
+      accessor: 'cobs.percent_kmers_found',
     },
   ];
   if (!data) return null;
 
   return (
     <section>
-      <h3>BIGSI Results</h3>
+      <h3>COBS Results</h3>
       <EMGTable
         cols={columns}
-        data={data.results as Record<string, unknown>[]}
+        data={data.data.results as Record<string, unknown>[]}
```

Comment:
data.data?

Reply:
Yup! One `data` for the name of the data object from the `useData` hook (which also contains things like `error`). Second `data` for the `data` object in the response from API (which might contain other things like `links`)

Reply:
got it, thanks!